.github/workflows/ci.yml

```diff
@@ -157,7 +160,7 @@ jobs:
 
       - name: "Upload to codecov.io"
         if: ${{ contains(env['RUN_ANALYZER'], 'cov') }}
-        uses: codecov/codecov-action@v1
+        uses: codecov/codecov-action@v3
```

Comment:
thank you, @vaind! so to summarize:

* We accept the non-SHA-pinned version tags for actions/* and getsentry/* because we trust them on all future versions of their actions.
* Someone verified `d9f34f8cd5cb3b3eb79b3e4b5dae3a16df499a70` of `codecov-action` so we trust this particular commit, but not necessarily future v3 releases.

Under these conditions, it should also be safe to enable dependabot on the `github-actions` package-ecosystem, right?